snuba/settings/__init__.py

```diff
@@ -29,9 +29,9 @@
 # Clickhouse Options
 CLICKHOUSE_MAX_POOL_SIZE = 25
 
-# The number of physical partitions that we will need to map resources to
-# for storage partitioning
-LOCAL_PHYSICAL_PARTITIONS = 1
```

Comment:
Even though this is not needed across all of storage sets, there would be a requirement to map logical partitions to a specific physical partitions based on storage set.

Reply:
I'll add the LOCAL_PHYSICAL_PARTITIONS back.